Library/Formula/ppss.rb

```diff
@@ -1,9 +1,9 @@
-require 'formula'
-
 class Ppss < Formula
-  homepage 'http://ppss.googlecode.com'
-  url 'https://ppss.googlecode.com/files/ppss-2.97.tgz'
-  sha1 '097dd068c16078ead8024551be6e69786f8ba533'
+  homepage "https://github.com/louwrentius/PPSS"
+  url "https://ppss.googlecode.com/files/ppss-2.97.tgz"
+  sha256 "25d819a97d8ca04a27907be4bfcc3151712837ea12a671f1a3c9e58bc025360f"
+
+  head "https://github.com/louwrentius/PPSS.git"
 
   def install
     bin.install "ppss"
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!